src/components/CardSections/QuickStart.jsx

```diff
@@ -13,9 +13,9 @@ export default function QuickStartSection({ title, className }) {
         to="/SDK/Explanation/installation"
       />
       <Card
-        title="⌨️ Encrypt and Decrypt Static Content"
-        description="Create your first Lit enabled application."
-        to="/SDK/Explanation/encryption"
+        title="👋 Hello World"
```

Comment:
I would keep the encrypt and decrypt static content example! This is the easiest starter for folks and still a core functionality that people are coming to Lit for.